src/Whoops/Run.php

```diff
@@ -395,6 +410,13 @@ private function writeToOutputNow($output)
         return $this;
     }
 
+    private static function isInternalErrorException($exception)
+    {
+        return class_exists('Error') &&
+            !is_subclass_of('Error', 'Exception') &&
```

Comment:
Shouldnt it just check for php7 and use the Error class then (which will always be available in php7 runtime)